doc/classes/ItemList.xml

```diff
@@ -115,6 +116,15 @@
 				Returns the metadata value of the specified index.
 			</description>
 		</method>
+		<method name="get_item_rect" qualifiers="const">
+			<return type="Rect2" />
+			<param index="0" name="idx" type="int" />
+			<param index="1" name="expand" type="bool" default="true" />
+			<description>
+				Returns the position and size of the item with the specified index, in the coordinate system of the [ItemList] node. If [param expand] is [code]true[/code] the last column expands to fill the rest of the row.
+				[b]Note:[/b] The returned value is not reliable if any changes has been made to the [ItemList] after it was last redrawn.
```

Comment:
This is a bit vague. ItemList will automatically redraw when any changes are made.
Maybe it should be something like `The returned value is unreliable if called right after modifying the [ItemList], before it redraws in the next frame.`

Reply:
Sure!